pennylane/capture/autograph/transformer.py

```diff
@@ -119,15 +119,66 @@ def get_cached_function(self, fn):
 def run_autograph(fn):
     """Decorator that converts the given function into graph form.
 
+    AutoGraph can be used in PennyLane's capture workflow to convert Pythonic control flow to PennyLane
+    native control flow. This requires the ``diastatic-malt`` package, a standalone fork of the AutoGraph
+    module in TensorFlow (`official documentation <https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/autograph/g3doc/reference/index.md>`_
+    ).
+
+    Args:
+        fn (Callable): the callable to be converted. This could be a function, a QNode, or another callable object.
+            For a QNode, the ``Qnode.func`` will be converted. For another callable object, a function calling the
+            object will be converted.
+
+    Returns:
+        The converted result (Callable): For a function, the converted function is returned directly.
+            For a QNode, the ``QNode.func`` will be updated with the converted version of ``func``.
+            For any other callable ``obj``, the returned function will be a converted version of
+            ``lambda *args, **kwargs: obj(*args, **kwargs)``
+
+    .. note::
+
+        There are some limitations and sharp bits regarding AutoGraph; to better understand
+        supported behaviour and limitations, see https://docs.pennylane.ai/en/stable/development/autograph.html
```

Comment:
Until this is actually published, this link doesn't work - I'm not well-versed enough in sphinx to know if there is a better way to link this. This link shows the rendered version:

https://xanaduai-pennylane--6678.com.readthedocs.build/en/6678/development/autograph.html

Reply:
```suggestion
        supported behaviour and limitations, see :doc:`/development/autograph`.
```
The best way to link docs in the PL docs is to use the `:doc:` role. It has the same syntax as `:func:` etc., but takes an absolute path (from the docs directory) or relative path (from the current file) to the corresponding rst file you want to link from.

This makes it independent of URL. In the stable docs it will generate a stable URL, and in the latest docs it will generate a latest URL.

Reply:
It will also work in PR previews

Reply:
That's super helpful, thanks!